core/src/test/java/org/apache/iceberg/rest/requests/TestFetchScanTasksRequest.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iceberg.rest.requests;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import org.junit.jupiter.api.Test;
+
+public class TestFetchScanTasksRequest {
+
+  @Test
+  public void nullAndEmptyCheck() {
+    assertThatThrownBy(() -> FetchScanTasksRequestParser.toJson(null))
+        .isInstanceOf(IllegalArgumentException.class)
+        .hasMessage("Invalid request: fetchScanTasks request null");
+
+    assertThatThrownBy(() -> FetchScanTasksRequestParser.fromJson((JsonNode) null))
+        .isInstanceOf(IllegalArgumentException.class)
+        .hasMessage("Invalid request: fetchScanTasks null");
+  }
+
+  @Test
```

Comment:
this should also be testing with an empty json. Additionally, this should test similar to https://github.com/apache/iceberg/blob/f19643a93f5dac99bbdbc9881ef19c89d7bcd3eb/core/src/test/java/org/apache/iceberg/rest/requests/TestCreateViewRequestParser.java#L51 where the json has some fields but the required fields are missing (in this particular case only `plan-task` is a required field). The same testing pattern should be applied to all other parsers

Reply:
Will fix this